test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/OpenApiHttpTriggerContextTests.cs

```diff
@@ -137,7 +161,7 @@ public async Task Given_Authorization_When_AuthorizeAsync_Invoked_Then_It_Should
         public async Task Given_Type_When_GetOpenApiSpecVersion_Invoked_Then_It_Should_Return_Result(string version, OpenApiSpecVersion expected)
         {
             var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
-            var context = new OpenApiHttpTriggerContext();
+            var context = new OpenApiHttpTriggerContext(null);
```

Comment:
If we change the constructor parameter to be optional, we don't need to change this part.